packages/zowe-explorer/src/trees/job/JobTree.ts

```diff
@@ -1061,13 +1061,6 @@ export class JobTree extends ZoweTreeProvider<IZoweJobTreeNode> implements Types
         const pollInterval = await this.showPollOptions(node.resourceUri);
 
         if (pollInterval === 0) {
-            Gui.showMessage(
```

Comment:
Curious why this message was removed - do we consider it to be superfluous?

Reply:
Yep, we had some discussions in the past about the redundancy of "cancelled dialogs" as they don't really mesh well with the VS Code UX. I encountered this one while testing polling with the changes from this PR, so I figured it was best to remove it.